esx_service/vmdk_ops_test.py

```diff
@@ -238,5 +238,6 @@ def test_failure(self):
         if not paths:
             logging.debug("Directory clean up - removing  %s", path)
             os.removedirs(path)
-    #suite = unittest.TestLoader().loadTestsFromTestCase(TestStringMethods)
-    #unittest.TextTestRunner(verbosity=2).run(suite)
+
+        # If the unittest failed, re-raise the error
+        raise
```

Comment:
Once raised which code handles the exception. Its best if all the tests or test scripts run thru as they a single test run tells us what all problems are there than if the test itself stops on the first exception.